src/trinerdi/data-structures/fast_line_container.cpp

```diff
@@ -1,4 +1,5 @@
 /**
+ * Name: Line container
```

Comment:
We have two `Line containers` now; we could name this `Fast line container` or `Sorted line container` or something